src/components/MainPage/MainPage.js

```diff
@@ -76,41 +77,60 @@ const MainPage = ({
-            up-to-date data and changes compared to last year. An additional section on crime
-            indicators will be added at a later date.
-            <br />
-            <i>
-              Last updated:{" "}
-              {activeTab === JAILS ? jailsLastUpdatedDate : correctionsLastUpdatedDate}
-            </i>
-          </p>
+          <>
+            <p className="MainPage__description">
+              The following is a broad overview of the corrections system in {stateName},
+              representing up-to-date data and changes compared to last year. An additional section
+              on crime indicators will be added at a later date.
+              <br />
+              <i>
+                Last updated:{" "}
+                {activeTab === JAILS ? jailsLastUpdatedDate : correctionsLastUpdatedDate}
+              </i>
+            </p>
+          </>
         )}
       </header>
-      <Tabs activeTab={activeTab} onTabChange={onActiveTabChange} />
-      {activeTab === CORRECTIONS && (
-        <Corrections
-          populationsChartData={populationsChartData}
-          prisonAdmissionsChartData={prisonAdmissionsChartData}
-          paroleRevocationsChartData={paroleRevocationsChartData}
-          probationRevocationsChartData={probationRevocationsChartData}
-          releasesChartData={releasesChartData}
-          flowDiagramData={flowDiagramData}
-          flowDiagramLastDate={flowDiagramLastDate}
-          flowDiagramPrevDate={flowDiagramPrevDate}
-          keyInsightsData={correctionsKeyInsightsData}
-          sourceData={correctionsSourceData}
-        />
-      )}
-      {activeTab === JAILS && (
-        <Jails
-          keyInsightsData={jailsKeyInsightsData}
-          countySelector={countySelector}
-          incarcerationRateChartData={incarcerationRateChartData}
-          incarcerationRateTopCountiesChartData={incarcerationRateTopCountiesChartData}
-          sourceData={jailsSourceData}
-        />
+      {!isNoData && (
+        <>
+          <Tabs activeTab={activeTab} onTabChange={onActiveTabChange} />
+          {activeTab === CORRECTIONS && (
+            <div className="MainPage__range">
+              <h3 className="MainPage__range-title">Data Aggregation Range</h3>
+              <p className="MainPage__description">
+                Use the control below to switch between{" "}
+                <strong>showing only monthly-aggregated data</strong> and showing{" "}
+                <strong>both monthly and annually-aggregated data</strong>.
+              </p>
+              <div className="MainPage__switch">
+                <Switch activeTab={activePane} onTabChange={onActivePaneChange} />
+                <p className="MainPage__switch-label">
+                  {activePane === ANNUAL
+                    ? `Only data that is aggregated monthly will be shown in the Key Insights and flow diagram below. Click the control to the left to see monthly data.`
```

Comment:
this is a pre-existing issue I believe but 'monthly' -> 'annual' for the first occurrence on this line

Reply:
fixed